src/api/spec/jobs/measurements_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe MeasurementsJob do
+  before do
+    rabbit_mock = Class.new do
+      attr_reader :metrics
+
+      def initialize
+        @metrics = {}
+      end
+
+      def metrics_sent
+        @metrics
+      end
+
+      def send_to_bus(category, data)
+        @metrics[category] = [] unless @metrics[category]
+        @metrics[category] << data
+      end
+    end
+    stub_const('RabbitMock', rabbit_mock)
+  end
```

Comment:
Wouldn't it be easier to mock `RabbitmqBus` like here?

https://github.com/openSUSE/open-build-service/blob/c961fe2c2841a16eab492b416f5f544a10b5dbe9/src/api/spec/controllers/webui/status_messages_controller_spec.rb#L118-L127